task1/src/redux/reducer/albumsReducer.js

```diff
@@ -0,0 +1,63 @@
+import { ADD_ALBUM_FAILURE, ADD_ALBUM_STARTED, ADD_ALBUM_SUCCESS, GET_ALBUMS_FAILURE, GET_ALBUMS_STARTED, GET_ALBUMS_SUCCESS, SET_ACTIVE_ALBUM, SET_ALBUMS } from "../const/const";
+
+const initialState = {
+    albums: [],
+    albumsLoaded: true,
+    albumAdded: true,
+    additionalAlbums: [],
+    albumsLoadError: null,
+    addAlbumError: null,
+    activeAlbumInfo: { albumId: null, photos: [] },
+}
+
+export default function albumReducer(state = initialState, action) {
+    switch (action.type) {
+        // get albums reducers
+        case GET_ALBUMS_STARTED:
+            return {
+                ...state,
+                albumsLoaded: false,
```

Comment:
more preferable to use something like `albumsLoading` here

Reply:
answered in the previous comment